lib/generators/stimulus_reflex/templates/app/javascript/channels/index.js.importmap.tt

```diff
@@ -0,0 +1,22 @@
+const registeredChannels = {}
+
+// Eager load all Action Cable channels registered beneath the `under` path in the import map
+export function eagerLoadChannelsFrom (under) {
```

Comment:
You should call the `eagerLoadChannelsFrom()` function in this file

Reply:
Great catch!